ams/ams-server/src/main/java/com/netease/arctic/ams/server/service/impl/DDLTracerService.java

```diff
@@ -303,6 +305,8 @@ public void syncDDl(ArcticTable arcticTable) {
 
     public void syncProperties(TableMetadata tableMetadata, ArcticTable arcticTable) {
       Table table = arcticTable.isKeyedTable() ? arcticTable.asKeyedTable().baseTable() : arcticTable.asUnkeyedTable();
+      PropertiesUtil.removeHiddenProperties(tableMetadata.getProperties(), ServerTableProperties.HIDDEN_INTERNAL);
```

Comment:
This map is an immutable map, can not execute remove operation

Reply:
fixed in new commit